### DISCUSSION TITLE
How I can know which version of Netherite backend we are running? (Typescript)

### DISCUSSION THREAD

**Discussion by XBeg9** [microsoft/durabletask-netherite#281] [General]
Hi Team, 

Am I missing something, you do have releases here, but is there a way to control which Netherite backend version I am running using host.json file? Thanks

**Comment by davidmrdavid** [comment]
Hi @XBeg9: the answer depends on what kind of app this is? Is this an Azure Functions app? If so, is this for C# in-process, C# isolated, JavaScript, Python, or something else?

**Reply by XBeg9** [reply]
It's Azure Functions app, javascript runtime.

**Reply by davidmrdavid** [reply]
That makes sense. If you're using JavaScript then, by default, the version of Netherite you use it automatically managed for you through the [extension bundles](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-register#extension-bundles) feature. Extension Bundles will periodically obtain the latest Netherite extension version for your app, but in practice it tends to be about a month delayed.


If you want to opt out of extension bundles and have direct control over the Netherite version you use, then you can have to follow the instructions [here](https://learn.microsoft.com/en-us/azure/azure-functions/functions-how-to-use-azure-function-app-settings?tabs=portal#manually-install-extensions). Using those instructions, you'll want to install this package in particular: https://www.nuget.org/packages/Microsoft.Azure.DurableTask.Netherite.AzureFunctions

**Reply by davidmrdavid** [reply]
Let me know if you need assistance on the specifics and I'll do my best to unblock you, but please do refer to the documentation first :). Thanks!
